src/Rotations.jl

```diff
@@ -1,5 +1,48 @@
+# we generate code in this module, so precompile where possible
+ VERSION >= v"0.4.0-dev+6521" && __precompile__(true)
+
 module Rotations
 
-# package code goes here
+using FixedSizeArrays
+using Quaternions  # TODO: Switch to the UnitQuaternions package? https://github.com/kam3k/UnitQuaternions.jl
+
+import Base.convert, Base.mean, Base.eltype, Base.isnan, Base.length, Base.call, Base.getindex, Base.inv, Base.*
+import Quaternions.Quaternion
+
+
+export
+
+        # Euler angle ordering 
+        EulerOrder,
+        EulerZXY,
```

Comment:
I find it amusing that we have hundreds of line of code generators and generated code for these, and yet only export `EulerZXY`... Not sure I have a better suggestion.